dumpling/export/dump_test.go

```diff
@@ -6,66 +6,15 @@ import (
 	"context"
 	"fmt"
 	"testing"
-	"time"
 
 	"github.com/DATA-DOG/go-sqlmock"
-	"github.com/pingcap/errors"
 	"github.com/pingcap/tidb/br/pkg/version"
 	tcontext "github.com/pingcap/tidb/dumpling/context"
 	"github.com/pingcap/tidb/parser"
 	"github.com/pingcap/tidb/util/promutil"
 	"github.com/stretchr/testify/require"
-	"golang.org/x/sync/errgroup"
 )
 
-func TestDumpBlock(t *testing.T) {
```

Comment:
Why this test is deleted?

Reply:
This test is used to check whether `dumpDatabases` can quit correctly and quickly when writerGoroutines exits. Maybe we should leave a similar test?

Reply:
revert it https://github.com/pingcap/tidb/pull/38625/commits/20617f44705c908911cb6485da293bde31f6dd82